src/interfaces/IMetaMorpho.sol

```diff
@@ -164,8 +164,6 @@ interface IMetaMorphoBase {
     function updateWithdrawQueue(uint256[] calldata indexes) external;
 
     /// @notice Reallocates the vault's liquidity so as to reach a given allocation of assets on each given market.
-    /// @notice The allocator can withdraw from any market, even if it's not in the withdraw queue, as long as the loan
-    /// token of the market is the same as the vault's asset.
```

Comment:
Why is this removed?

Reply:
Because it's false now, see the description of the PR (first point)